kimchi/src/prover.rs

```diff
@@ -1101,7 +1101,7 @@ where
 
         //~ 1. Evaluate the same polynomials without chunking them
         //~    (so that each polynomial should correspond to a single value this time).
-        let evals = {
+        let evals: ProofEvaluations<PointEvaluations<G::ScalarField>> = {
```

Comment:
Why is adding the explicit type needed now?

Reply:
I guess readability. `evals` sounds like it can be any evaluations -- `ProofEvaluations<G::ScalarField>` or `PointEvaluations<somethingelse>`. Ultimately I don't know -- I probably added it because I found it confusing... But I don't think explicit typing hurts here, does it?